### ISSUE TITLE
`query` and `run` only take string parameters

### ISSUE THREAD

**Issue by chriswep** [capacitor-community/sqlite#110]
When using numbers or `null` in a query (or in `run()`) the plugin returns with the error "Must provide an Array of string" (3.0.0-beta-9). Replacing the type with `Any` (for testing) in the relevant places in the iOS code seemed to work fine for my application (via TypeORM plugin). I'm don't know the reasoning for the `String` type here but i would suggest that at least `number` and `null` are needed in a lot of SQLite use cases.

@jepiqueau sorry for adding so many issues. i was working on the TypeORM plugin last night and those were the issues i encountered. Thanks for the hard work!

**Comment by jepiqueau** [comment]
@chriswep On the run command the type for values is any
``` 
export interface capSQLiteRunOptions {
  /**
   * The database name
   */
  database?: string;
  /**
   * A statement
   */
  statement?: string;
  /**
   * A set of values for a statement
   */
  values?: any[];
}
``` 
see `https://github.com/jepiqueau/angular-sqlite-app-starter/blob/master/src/app/test2dbs/test2dbs.page.ts`
On contrary the query required an Array of strings

I understood that you started to work on the TypeORM plugin in view of the number of questions. But it is the TypeORM plugin which must adapt to the @capacitor-community/sqlite plugin and not the reverse otherwise i could have done it which was not advise by other developers using the plugin in view of the complexity of the TypeORM plugin.

**Comment by chriswep** [comment]
I don't expect the plugin to adapt to TypeORM. My impression is that this is a bug though (of course i might be wrong).

You are right about the run method - i probably wrongly thought so, because i had to use the query method for that as well (#111).

The question remains for me concerning the query method. Is there a reason to not allow number or null here?

**Comment by jepiqueau** [comment]
@chriswep solved in 3.0.0-beta.11
